src/current/_includes/releases/v24.2/v24.2.4.md

```diff
@@ -0,0 +1,95 @@
+## v24.2.4
+
+Release Date: October 17, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
```

Comment:
```suggestion
{% include releases/new-release-downloads-docker-image.md release=include.release %}

```
I have reported this to the Tools team. It can cause some display issues depending on what comes after the include.